src/Client.js

```diff
@@ -182,13 +179,13 @@ function Client(options) {
     headers: {},
     fetch: undefined,
     queryTimeout: null,
-    http2SessionIdleTime: http2SessionIdleTime.value,
+    http2SessionIdleTime: 500,
     checkNewVersion: false,
   })
 
-  if (http2SessionIdleTime.shouldOverride) {
-    options.http2SessionIdleTime = http2SessionIdleTime.value
-  }
+  options.http2SessionIdleTime = getHttp2SessionIdleTime(
+    options.http2SessionIdleTime
+  )
```

Comment:
If we plug it in on line 182 it can be overwitten as that function is just stuffing in defaults if things haven't been explicitly set. I agree though the way I'm doing it now is a little weird. I'll see if I can come up with something better.